api/app/tests/apps/api/v1/test_private_questions_endpoint.py

```diff
@@ -0,0 +1,89 @@
+from django.contrib.auth.models import Permission
+from rest_framework import status
+
+from signals.apps.signals.factories import QuestionFactory
+from signals.apps.signals.models import Question
+from tests.test import SIAReadWriteUserMixin, SignalsBaseApiTestCase
+
+
+class TestPrivateExpressionEndpoint(SIAReadWriteUserMixin, SignalsBaseApiTestCase):
+    list_endpoint = '/signals/v1/private/questions/'
+    detail_endpoint = '/signals/v1/private/questions/{pk}/'
+
+    def setUp(self):
+        self.questions_read = Permission.objects.get(
+            codename='sia_question_read'
+        )
+        self.questions_write = Permission.objects.get(
+            codename='sia_question_write'
+        )
+        self.sia_read_write_user.user_permissions.add(self.questions_read)
+        self.sia_read_write_user.user_permissions.add(self.questions_write)
+
+        self.question = QuestionFactory.create()
+
+    def _create_question(self, key, field_type, required):
+        data = {
+            'key': key,
+            'field_type': field_type,
+            'required': required,
+            'meta': {
+                'label': 'test question',
+                'values': {
+                    'key': 'value'
+                }
+            }
+        }
+
+        response = self.client.post(
+            self.list_endpoint, data=data, format='json'
+        )
+        return response
+
+    def test_get_detail(self):
+        self.client.force_authenticate(user=self.sia_read_write_user)
+        response = self.client.get(self.detail_endpoint.format(pk=self.question.pk))
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_post(self):
+        self.client.force_authenticate(user=self.sia_read_write_user)
+        response = self._create_question(
+            key='testkey',
+            field_type=Question.PLAIN_TEXT,
+            required=True
+        )
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+    def test_patch(self):
+        self.client.force_authenticate(user=self.sia_read_write_user)
+
+        data = {
+            'key': 'newkey',
```

Comment:
We should discuss this patching questions. There are some implications with regards to reporting / analysis of complaints/meldingen. Questions should have unique key, we have that in the `Question` model --- so that is good. However, by patching a question it becomes another question for reporting purposes and should have another key (which you could leave out of the patch).